apps/client/src/app/challenges/add/page.tsx

```diff
@@ -0,0 +1,26 @@
+import { Anchor, Box, Breadcrumbs, Divider, Title } from '@mantine/core';
+
+import ChallengesForm from '@/components/form/ChallengesForm';
+
+const items = [
+  { title: 'Challenges', href: '/challenges' },
+  { title: 'Add Challenge', href: '#' }
+].map((item, index) => (
+  <Anchor href={item.href} key={index}>
+    {item.title}
+  </Anchor>
+));
```

Comment:
Can we do the following?

```suggestion
const items = [
  { title: 'Challenges', href: '/challenges' },
  { title: 'Add Challenge', href: '#' }
];

return (
  <Breadcrumbs>
    {items.map((item, index) => (
      <Anchor href={item.href} key={index}>
        {item.title}
      </Anchor>
    ))}
  </Breadcrumbs>
)
```

Reply:
Done